plugin/distribution/types.go

```diff
@@ -7,6 +7,9 @@ import "errors"
 // ErrUnSupportedRegistry indicates that the registry does not support v2 protocol
 var ErrUnSupportedRegistry = errors.New("Only V2 repositories are supported for plugin distribution")
 
+// ErrUnSupportedMediaType indicates we are pulling content that's not a plugin
+var ErrUnSupportedMediaType = errors.New("Content is not a plugin")
```

Comment:
nit: `Unsupported` instead of `UnSupported`.

Also, error text should be lowercase.